pkg/relayer/proxy/proxy_test.go

```diff
@@ -679,3 +723,20 @@ func sendRequestWithSuccessfulReply(
 
 	return testproxy.MarshalAndSend(test, proxiedServices, "server1", "service1", req)
 }
+
+func sendRequestWithCustomSessionHeight(
+	requestSessionBlockHeight int64,
+) func(t *testing.T, test *testproxy.TestBehavior) (errCode int32, errorMessage string) {
+	return func(t *testing.T, test *testproxy.TestBehavior) (errCode int32, errorMessage string) {
+		req := testproxy.GenerateRelayRequest(
+			test,
+			appPrivateKey,
+			"service1",
```

Comment:
consider moving this "magic stirng" into a local constant. We reuse it in a few places.